docs/PeggyForRemix.sol

```diff
@@ -0,0 +1,733 @@
+pragma solidity ^0.5.0;
+

```

Comment:
this file seems to be a copy paste of Peggy.sol, is it necessary?

Reply:
See https://github.com/cosmos/peggy/pull/51#discussion_r306593301